pkg/apis/pingcap/v1alpha1/tikv_config.go

```diff
@@ -194,11 +194,11 @@ type TiKVRaftDBConfig struct {
 // +k8s:openapi-gen=true
 type TiKVSecurityConfig struct {
 	// +optional
-	CAPath string `json:"ca_path,omitempty" toml:"ca_path,omitempty"`
+	CAPath string `json:"ca-path,omitempty" toml:"ca-path,omitempty"`
 	// +optional
-	CertPath string `json:"cert_path,omitempty" toml:"cert_path,omitempty"`
+	CertPath string `json:"cert-path,omitempty" toml:"cert-path,omitempty"`
 	// +optional
-	KeyPath string `json:"key_path,omitempty" toml:"key_path,omitempty"`
+	KeyPath string `json:"key-path,omitempty" toml:"key-path,omitempty"`
```

Comment:
There are many other keys are connected with `_`, do they should be change to `-`? @aylei PTAL